src/exception/exception-manifest/util.ts

```diff
@@ -157,15 +159,26 @@ export const mapEntityExceptionRecords = (programId: string, donors: DeepReadonl
 	return entityRecord;
 };
 
-export const sortExceptionRecordsBySubmitterId = (
+function baseExceptionSort<Exception>(
```

Comment:
why the change to use the `function` keyword?

Reply:
Oh haha this was just following TS Docs example
It could be changed to:

`export const baseExceptionSort = <Exception>(
	first: Exception,
	next: Exception,
	key: keyof Exception,
): number => {
	return first[key] === next[key] ? 0 : first[key] > next[key] ? 1 : -1;
};`